SimpleSDMPolygons/src/providers/openstreetmap.jl

```diff
@@ -0,0 +1,30 @@
+const OpenStreetMapDatasets = Union{
+    Countries
+}
+
+provides(::Type{OpenStreetMap}, ::Type{T}) where {T<:OpenStreetMapDatasets} = true 
+downloadtype(::PolygonData{OpenStreetMap,T}) where T = _File
+filetype(::PolygonData{OpenStreetMap,T}) where T = _GeoJSON
+root(::PolygonData{OpenStreetMap,Countries}) = "https://polygons.openstreetmap.fr/get_geojson.py?id="
+
+function _query_osm_id(place)
+    url_place = HTTP.URIs.escapeuri(place)
+    query = "https://nominatim.openstreetmap.org/search?q=$(url_place)&format=geojson"
+    id = first(GeoJSON.read(Downloads.download(query)).osm_id)
+    return id
+end
+
+
+function source(data::PolygonData{OpenStreetMap, Countries}; country = "Canada")    
```

Comment:
`country` as a keyword is misleading as OSM provides access to all kind of places, including super-national areas like mountain ranges -- I would use `place` and describe a type for `Place` as well

Reply:
done